test/integration/without-pam/ssh_password_selinux_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe command('getenforce') do
+  its(:exit_status) { should eq 0 }
+  its(:stdout) { should include 'Enforcing' }
+  its(:stderr) { should eq '' }
+end
+
+describe command('semodule -l | grep "ssh_password"') do
+  its(:exit_status) { should eq 0 }
+  its(:stdout) { should include 'ssh_password' }
+  its(:stderr) { should eq '' }
+end
```

Comment:
It think it would be better to add the test to the ssh-baseline repo, that we have all tests there. What do you think?